core-sdk-samples/higgs-shop-sample-app/app/src/main/kotlin/com/mparticle/example/higgsshopsampleapp/LoggingCustomKit.kt

```diff
@@ -0,0 +1,88 @@
+package com.mparticle.example.higgsshopsampleapp
+
+import android.content.Context
+import android.util.Log
+import com.mparticle.MPEvent
+import com.mparticle.internal.CoreCallbacks.KitListener
+import com.mparticle.kits.KitIntegration
+import com.mparticle.kits.KitIntegration.EventListener
+import com.mparticle.kits.MPSideloadedKit
+import com.mparticle.kits.ReportingMessage
+import java.lang.Exception
+
+class LoggingCustomKit : MPSideloadedKit(), KitListener, EventListener {
```

Comment:
Please update the log messages to correspond to the method name and log the appropriate message, if one exists.

For example, `logError` has `"CUSTOM_KIT event log with name "` as the log message when it should be something like `"CUSTOM_KIT error log with message ${message}"`, etc. 

Since this is an example project specifically for new mParticle customers or customers first testing this feature, these log messages could be very confusing.